src/utils/utils.py

```diff
@@ -163,7 +162,7 @@ def process_sentence(sent, max_seq_len, sos_tok=SOS_TOK, eos_tok=EOS_TOK):
     max_seq_len -= 2
     assert max_seq_len > 0, "Max sequence length should be at least 2!"
     if isinstance(sent, str):
-        return [sos_tok] + TOKENIZER.tokenize(sent)[:max_seq_len] + [eos_tok]
+        return [sos_tok] + TOKENIZER(sent)[:max_seq_len] + [eos_tok]
```

Comment:
Does the new package change the interface? If so, we need to change this everywhere else the tokenizer is called. Can you also verify that the new tokenizer exactly matches the old one?